src/components/application_manager/include/application_manager/application.h

```diff
@@ -639,6 +640,18 @@ class Application : public virtual InitialApplicationData,
    */
   virtual UsageStatistics& usage_report() = 0;
 
+  /**
+   * @brief Access to HelpPromptManager interface
+   * @return object for Handling VR help
+   */
+  virtual HelpPromptManager& help_prompt_manager() = 0;
```

Comment:
Duplicate function, one of these should be deleted.

Reply:
@jacobkeeler removed in 1c57695b83f1d680a8ea23513378e6c6248210da